docs/edgecases.md

```diff
@@ -52,99 +54,118 @@ building
+ModuleNotFoundError: No module named 'setuptools'
+
+error (ignored): error: cannot unlink '/tmp/nix-build-python3.10-django-4.1.3.drv-1/Django-4.1.3': Directory not empty
+error: 1 dependencies of derivation '/nix/store/hz4b87s99s1lwiz2m0vwilhlh6rlfj64-python3-3.10.8-env.drv' failed to build
+error: 1 dependencies of derivation '/nix/store/60gaxbdf11lhaj9xg50cf0rr6x5v8v1z-nix-shell-env.drv' failed to build
 ```
+
+As you can see on the fourth last line it's missing `setuptools` which in this case is our missing `PACKAGENAME`.
+
 </details>
 
-In order to be able to build `extralib` we should modify our nix definition as follows:
+In order to be able to build `django-floppyforms` we should modify our nix definition as follows:
+
+``` nix
+poetry2nix.mkPoetryApplication {
+  projectDir = ./.;
+  overrides = poetry2nix.defaultPoetryOverrides.extend
+    (self: super: {
+      django-floppyforms = super.django-floppyforms.overridePythonAttrs
+      (
+        old: {
+          buildInputs = (old.buildInputs or [ ]) ++ [ super.setuptools ];
+        }
+      );
+    });
+}
 ```
+
+This override will instruct the underlying build logic to include the additional build dependency into the inputs of `django-floppyforms`.
+It might help to know that you don't need to use the full package name like `python39Packages.setuptools` but can just use `setuptools` directly.
+Be aware that the package names are normalized according to PEP-517, so something like `flit_scm` becomes `flit-scm`.
+
+It can happen that you need multiple overrides for your project, just work through one after the other and your project should build at the end.
+Your file might then look something like this:
+
+``` nix
 poetry2nix.mkPoetryApplication {
   projectDir = ./.;
-  overrides = poetry2nix.defaultPoetryOverrides.extend ( self: super: {
-    extralib = super.extralib.overridePythonAttrs (
-      old: {
-        buildInputs = (old.buildInputs or [ ]) ++ [ self.poetry ];
-      }
-    );
-  });
+  overrides = poetry2nix.defaultPoetryOverrides.extend
+    (self: super: {
+      first-dependency = super.first-dependency.overridePythonAttrs
+      (
+        old: {
+          buildInputs = (old.buildInputs or [ ]) ++ [ super.buildtools ];
+        }
+      );
+      second-dependency = super.second-dependency.overridePythonAttrs
+      (
+        old: {
+          buildInputs = (old.buildInputs or [ ]) ++ [ super.pdm ];
+        }
+      );
+    });
 }
 ```
-This override will instruct underlying build logic to include additional build dependency into inputs of `extralib`.
+
+We recommend that you contribute your changes to `poetry2nix` so that other users can profit from them as well.
+The file with the upstream overrides can be found under the following link. It is a simple JSON file which contains the overrides in an array and sorted in alphabetical order. https://github.com/nix-community/poetry2nix/blob/master/overrides/build-systems.json
```

Comment:
The link could just be inlined, as in

> We recommend that you contribute your changes to [build-systems.json](https://github.com/nix-community/poetry2nix/blob/master/overrides/build-systems.json) so that other users can profit from them as well.

Reply:
Implemented.